docs/requirements-doc.txt

```diff
@@ -0,0 +1,4 @@
+jupyter-book>=1.0.0
+sphinx>5.0.0
+sphinx_inline_tabs
+ghp-import
```

Comment:
I don't find any usage of `ghp-import` either.

Reply:
added here: https://github.com/recommenders-team/recommenders/pull/2051/commits/1aa1945d2d3190e053fb47c529fc58db7ebe3a65